library/Mockery/Generator/MockConfiguration.php

```diff
@@ -467,6 +498,14 @@ protected function getAllMethods()
             $methods = array_merge($methods, $class->getMethods());
         }
 
+        foreach ($this->getTargetTraits() AS $trait) {
+            foreach ($trait->getMethods() as $method) {
+                if ($method->isAbstract()) {
```

Comment:
It's been a while since I last looked at mockery's internals, but shouldn't you call `isTrait` here? If not, why do we need the `isTrait` method? I don't see it called anywhere.